### PR TITLE
Add container description

### PR DESCRIPTION
**Pull request by NathanielBennett**
Add a description to containers - where configured. Depend on: https://github.com/guardian/facia-scala-client/pull/119

container-description

### CODE REVIEW

File: common/app/layout/Front.scala [hunk 1]
```diff
@@ -193,7 +193,10 @@ object FaciaContainer {
       case MostPopular => ContainerCommercialOptions.empty
       case _ => ContainerCommercialOptions.fromConfig(config.config)
     },
-    None,
+    config.config.description match {
```
Comment (janua) [inline common/app/layout/Front.scala]: Could just `map` `DescriptionMetaHeader.apply` here?

File: project/Dependencies.scala [hunk 2]
```diff
@@ -18,8 +18,8 @@ object Dependencies {
   val crosswordsApiClient = "com.gu" %% "crosswords-api-client" % "0.5"
   val dfpAxis = "com.google.api-ads" % "dfp-axis" % "2.0.0"
   val exactTargetClient = "com.gu" %% "exact-target-client" % "2.24"
-  val faciaFapiScalaClient = "com.gu" %% "fapi-client" % "0.49"
-  val faciaScalaClient = "com.gu" %% "facia-json" % "0.49"
+  val faciaFapiScalaClient = "com.gu" %% "fapi-client" % "0.50-SNAPSHOT"
+  val faciaScalaClient = "com.gu" %% "facia-json" % "0.50-SNAPSHOT"
```
Comment (janua) [inline project/Dependencies.scala]: Of course you'll want to update this when it gets released!

File: onward/app/controllers/SeriesController.scala [hunk 3]
```diff
@@ -56,7 +56,8 @@ object SeriesController extends Controller with Logging with Paging with Executi
     val componentId = Some("series")
     val displayName = Some(series.displayName)
     val properties = FrontProperties(series.tag.description, None, None, None, false, None)
-    val header = Option(SeriesDescriptionMetaHeader(series.tag.description))
+    val header = series.tag.description map { description => DescriptionMetaHeader(description) }
```
Comment (janua) [inline onward/app/controllers/SeriesController.scala]: `map`?

**Comment by janua** [comment]
Can we merge in master again? I think there might be conflicts outside of `Dependencies.scala`, but if not then it's no problem really.

File: onward/app/controllers/SeriesController.scala (hunk 3 above)
Comment (NathanielBennett) [inline onward/app/controllers/SeriesController.scala]: What is the problem?

**Comment by piuccio** [comment]
looks good to me

File: onward/app/controllers/SeriesController.scala (hunk 3 above)
Comment (janua) [inline onward/app/controllers/SeriesController.scala]: Oops. I meant `map(DescriptionMetaHeader)`

**Pull request merged**